hack/vmimage.sh

```diff
@@ -3,12 +3,19 @@
 cleanup() {
   set +e
 
+  if [[ -e /var/run/secrets/kubernetes.io ]]; then
```

Comment:
Notification is not part of a cleanup. Can we move this check into a separate function-trap?